tensorflow/workspace.bzl

```diff
@@ -228,6 +239,29 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
       strip_prefix = "protobuf-2b7430d96aeff2bb624c8d52182ff5e4b9f7f18a",
   )
 
+  # We need to import the protobuf library under the names com_google_protobuf
+  # and com_google_protobuf_cc to enable proto_library support in bazel.
+  # Unfortunately there is no way to alias http_archives at the moment.
+  native.http_archive(
+      name = "com_google_protobuf",
```

Comment:
It's needed by the ops research tools: they have a protobuf_library bazel rule, which requires the workspace to import the protobuf code under the names com_google_protobuf and com_google_protobuf_cc.